### DISCUSSION TITLE
Discuss kubernetes extension for potential addition to OVAL 6.0 or removal of branch as obsolete?

### DISCUSSION THREAD

**Discussion by vanderpol** [OVAL-Community/OVAL#198] [General]
In cleaning up the branches of the OVAL repository, I came across the x-kubernetes branch created 5 years ago by @wmunyan.  https://github.com/OVAL-Community/OVAL/tree/x-kubernetes  

Is this something that others have adopted?  Should we discuss adding this to OVAL 6.0, or should it be deleted? 

Looking for feedback on others adoption of it @solind, @A-Biggs others?

**Comment by wmunyan** [comment]
Best information I have is that those tests were implemented in CIS-CAT, but I don't think that any benchmarks include any of the tests.

**Comment by A-Biggs** [comment]
I was not aware of that schema/branch, so our engine hasn't adopted it.  In say that we are getting requests with regards to kurernetes support, so I think it is worth trying to revive this, as I can see it being useful.

**Comment by vanderpol** [comment]
@A-Biggs I'll log an issue, and create a branch based on the new oval6-develop branch and assign it to you.  If you are able to test it out, or make refinements, that would be great just let the oval board know when/if it's ready for others to test and or merge into oval6-develop.
